### PR TITLE
Incoming webhook c# saample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/incoming-webhook/csharp/IncomingWebhook/ClientApp/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (AbhijeetThakur-msft) [inline samples/incoming-webhook/csharp/IncomingWebhook/ClientApp/README.md:1]: we can remove this Readme file

File: samples/incoming-webhook/csharp/IncomingWebhook/ClientApp/src/components/configure-message.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+﻿import React from 'react';
+import * as microsoftTeams from "@microsoft/teams-js";
+import { Flex, Button, Text, Input } from '@fluentui/react-northstar'
+import { SendIcon } from '@fluentui/react-icons-northstar'
+import "./configure-message.css"
+import { DEFAULT_CARD_PAYLOAD } from "./card-constant"
+import { sendCard } from './services/send-card-data.service';
+import Editor from '@monaco-editor/react';
+
```
Comment (AbhijeetThakur-msft) [inline samples/incoming-webhook/csharp/IncomingWebhook/ClientApp/src/components/configure-message.tsx:9]: try to add comments to the methods

File: samples/incoming-webhook/csharp/IncomingWebhook/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseDefaultFiles();
+            app.UseStaticFiles();
+            app.UseSpaStaticFiles();
+            app.UseWebSockets();
+            app.UseRouting();
+            app.UseAuthorization();
+            //app.UseMvc();
```
Comment (AbhijeetThakur-msft) [inline samples/incoming-webhook/csharp/IncomingWebhook/Startup.cs:62]: Please remove commented code if not required

File: samples/incoming-webhook/csharp/README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+languages:
+- csharp
+extensions:
+  contentType: samples
+  createdDate: "04/1/2022 2:36:57 PM"
+---
+
+# Incoming webhook
+
+This sample illustrates usage of incoming webhook.
```
Comment (AbhijeetThakur-msft) [inline samples/incoming-webhook/csharp/README.md:17]: Please add more details, like the editor part and action usage.

File: samples/incoming-webhook/csharp/README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  ```bash
+  # run ngrok locally
+  ngrok http -host-header=localhost 3978
+  ```
+
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+2) Clone the repository
```
Comment (AbhijeetThakur-msft) [inline samples/incoming-webhook/csharp/README.md:48]: The numbering should start with 1)

File: samples/incoming-webhook/csharp/README.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
+﻿---
+page_type: sample
```
Comment (AbhijeetThakur-msft) [inline samples/incoming-webhook/csharp/README.md:2]: this format is broken, please check once.
image

**Review by AbhijeetThakur-msft — APPROVED** [review]

File: samples/incoming-webhook/csharp/README.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  - File -> Open -> Project/Solution
+  - Navigate to `IncomingWebhook` folder
+  - Select `IncomingWebhook.csproj` file
+  - Press `F5` to run the project
+
+8) Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
+   - Go to Microsoft Teams. From the lower left corner, select Apps
+   - From the lower left corner, choose Upload a custom App
+   - Go to your project directory, the ./AppPackage folder, select the zip folder, and choose Open.
+   - Select Add to meeting in the pop-up dialog box. Your app is uploaded to Teams.
```
Comment (Wajeed-msft) [inline samples/incoming-webhook/csharp/README.md:94]: Please correct  this. This sample is not supported for meeting scenarios.

File: samples/incoming-webhook/csharp/README.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  ```bash
+  # run ngrok locally
+  ngrok http -host-header=localhost 3978
+  ```
+
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Create a incoming webhook. [Create incoming webhooks](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook)
```
Comment (Wajeed-msft) [inline samples/incoming-webhook/csharp/README.md:48]:
```suggestion
1) Create a incoming webhook. [Create incoming webhooks](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook). Keep this webhook URL handy while running the sample. 
```

File: samples/incoming-webhook/csharp/README.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+    # ngrok http -host-header=rewrite 3978
+    ```
+5) Modify the `manifest.json` in the `/AppPackage` folder and replace the following details
+   - `<<App-ID>>` with some unique GUID   
+   - `<<BASE-URL>>` with your application's base url, e.g. https://1234.ngrok.io
+   - `<<VALID DOMAIN>>` with your app domain e.g. *.ngrok.io
+
+6) Zip the contents of `AppPackage` folder into a `manifest.zip`, and use the `manifest.zip` to deploy in app store or add to Teams.
+
+7) Run the app from a terminal or from Visual Studio, choose option A or B.
```
Comment (Wajeed-msft) [inline samples/incoming-webhook/csharp/README.md:73]: After step 7, let's add one or two steps which talk about trying out the sample.

File: samples/incoming-webhook/csharp/README.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+   - Go to Microsoft Teams. From the lower left corner, select Apps
+   - From the lower left corner, choose Upload a custom App
+   - Go to your project directory, the ./AppPackage folder, select the zip folder, and choose Open.
+   - Select Add to meeting in the pop-up dialog box. Your app is uploaded to Teams.
+
+## Features of this sample
+
+## Further reading
+
+- [Bot Framework Documentation](https://docs.botframework.com)
```
Comment (Wajeed-msft) [inline samples/incoming-webhook/csharp/README.md:100]: Let's link to Connector sections instead of Bot.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**